sites/website/src/docs/fast-foundation/getting-started.md

```diff
@@ -6,22 +6,29 @@ custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website
 This document presumes you have [NodeJS](https://nodejs.org/) installed as well as some way to run an HTTP server locally, such as [http-server](https://www.npmjs.com/package/http-server).
 
 ## Quick-start
+
 This section will walk through the quickest way to get up and running using FAST components.
 
 ### Add the script
-Next, add the `fast-components.js` script file to your HTML. This pre-bundled script contains all components and dependencies. Also be sure to add [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to the script element.
+
+First, add the `fast-components.js` script file to your HTML. This pre-bundled script contains all components and dependencies. Also be sure to add [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to the script element.
 
 ```html
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <script src="https://static.fast.design/assets/scripts/packages/fast-components.js" type="module"></script>
+        <script src="...TODO..." type="module"></script>
     </head>
     <!-- ... -->
 </html>
 ```
 
+:::note
+If you are looking to leverage a tool like Webpack, see our [Webpack Guide](./webpack) to learn how to properly get set up.
+:::
+
 ### Add the Design System Provider
+
 The [Design System Provider](fast-foundation/fast-design-system-provider.md) will provide design information to child FAST components. 
 
 By default, the background color and text color won't be set by the `fast-design-system-provider` ([but there will be a solution soon](https://github.com/microsoft/fast-dna/issues/3213)), so you'll want to apply the CSS *background-color* and *color* properties.
```

Comment:
For any coming soon features, do we have a link to github issue we could add in case people want to follow along?

Reply:
This is the only "coming soon" I'm aware of. The link points to the proposed feature, so I think we're good.